modules/nf-core/gatk4/cnnscorevariants/main.nf

```diff
@@ -34,14 +34,14 @@ process GATK4_CNNSCOREVARIANTS {
     def architecture = architecture ? "--architecture $architecture" : ""
     def weights = weights ? "--weights $weights" : ""
 
-    def avail_mem = 3
```

Comment:
version bump missing here

Reply:
fixed, the docker image wasn't available at the time